CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Not released
 
+## 0.14.1
+- Add `#order` method to active query that accepts arguments in several formats ( symbol, string that has raw soql) (https://github.com/Beyond-Finance/active_force/pull/58)
+  - By default, the order is ascending. For descending order, you can map the column name symbol to :desc
+  - use-cases: User.order(:name), User.order(email: :desc),  User.order(:name, email: :desc), User.order('name')
+
```

Comment:
Remove line 7&8.  Just need the info on line 6.

Feel free to add info to README.md if you feel additional documentation is necessary, but since this pretty much follows the AR implementation, probably not necessary.

Reply:
Got it, Would update them